### PR TITLE
Updated the range keys to be values for completion

### PR DESCRIPTION
**Pull request by jaikoo**
Had to nest the results in an array and switch the keys for the ranges so that they all sat under the same key 'range' so that I can push the work of grouping to the keen.io visualisation library.

### CODE REVIEW

**Comment by threedaymonk** [comment]
Seems reasonable, except that it no longer indicates the difference between exclusive and inclusive ranges: which bucket does 20 sit in? (That probably doesn't matter if the scores are continuous rather than discrete. I can't remember.)

File: spec/services/metrics_publisher_spec.rb [hunk 1]
```diff
@@ -31,10 +31,24 @@
     )
     expect(recipient).to receive(:publish).
       with(:completion, a_hash_including(
-        '[0,20)'  => 10,
-        '[20,50)' => 20,
-        '[50,80)' => 30,
-        '[80,100]' =>  5
+        'completions_by_range' => [
+          {
+            'range' => '0 ≼ n < 20',
```
Comment (threedaymonk) [inline spec/services/metrics_publisher_spec.rb]: Should be `U+2264 LESS-THAN OR EQUAL TO`: ≤

**Comment by threedaymonk** [comment]
Looks good. Can you rebase and squash?

**Comment by jaikoo** [comment]
Yup, will do. Thanks.

File: app/services/metrics_publisher.rb [hunk 2]
```diff
@@ -19,13 +19,14 @@ def profiles_report
...
+      { 'range' => range_inclusion_as_string(range),
+        'total' => total }
     end
     report
   end
 
-  def range_to_string(range)
-    '[%d,%d%s' % [range.begin, range.end, range.exclude_end? ? ')' : ']']
+  def range_inclusion_as_string(range)
+    range.end == 100 ? "#{range.begin} ≤ n ≤ 100" : "#{range.begin} ≤ n < #{range.end}"
```
Comment (threedaymonk) [inline app/services/metrics_publisher.rb]: You could change `range.end == 100` to `range.exclude_end?` and swap the branches. Then it doesn't care about any external information.

**Pull request closed without merging**